dotCMS/src/main/java/com/dotcms/saml/SamlConfigurationServiceImpl.java

```diff
@@ -0,0 +1,318 @@
+                + property.getPropertyName() + "' to '" + value + "'");
+
+        switch (property) {
+
+            case DOTCMS_SAML_ASSERTION_RESOLVER_HANDLER_CLASS_NAME:
+                defaultParams.setDotcmsSamlAssertionResolverHandlerClassName(value);
+                break;
+            case DOTCMS_SAML_AUTHN_COMPARISON_TYPE:
+                defaultParams.setDotcmsSamlAuthnComparisonType(value);
+                break;
+            case DOTCMS_SAML_AUTHN_CONTEXT_CLASS_REF:
+                defaultParams.setDotcmsSamlAuthnContextClassRef(value);
+                break;
+            case DOTCMS_SAML_BINDING_TYPE:
+                defaultParams.setDotcmsSamlBindingType(value);
+                break;
+            case DOTCMS_SAML_BUILD_ROLES:
+                defaultParams.setDotcmsSamlBuildRoles(value);
+                break;
+            case DOTCMS_SAML_FORCE_AUTHN:
+                defaultParams.setDotcmsSamlForceAuthn(Boolean.parseBoolean(value));
+                break;
+            case DOTCMS_SAML_IDENTITY_PROVIDER_DESTINATION_SLO_URL:
+                defaultParams.setDotcmsSamlIdentityProviderDestinationSloUrl(value);
+                break;
+            case DOTCMS_SAML_IDENTITY_PROVIDER_DESTINATION_SSO_URL:
+                defaultParams.setDotcmsSamlIdentityProviderDestinationSsoUrl(value);
+                break;
+            case DOTCMS_SAML_INCLUDE_ROLES_PATTERN:
+                defaultParams.setDotcmsSamlIncludeRolesPattern(value);
+                break;
+            case DOTCMS_SAML_IS_ASSERTION_ENCRYPTED:
+                defaultParams.setDotcmsSamlIsAssertionEncrypted(Boolean.parseBoolean(value));
+                break;
+            case DOTCMS_SAML_IS_LOGOUT_NEED:
+                defaultParams.setDotcmsSamlIsLogoutNeeded(Boolean.parseBoolean(value));
+                break;
+            case DOTCMS_SAML_NAME_ID_POLICY_FORMAT:
+                defaultParams.setDotcmsSamlNameIdPolicyFormat(value);
+                break;
+            case DOTCMS_SAML_OPTIONAL_USER_ROLE:
+                defaultParams.setDotcmsSamlOptionalUserRole(value);
+                break;
+            case DOTCMS_SAML_POLICY_ALLOW_CREATE:
+                defaultParams.setDotcmsSamlPolicyAllowCreate(Boolean.parseBoolean(value));
+                break;
+            case DOTCMS_SAML_PROTOCOL_BINDING:
+                defaultParams.setDotcmsSamlProtocolBinding(value);
+                break;
+            case DOTCMS_SAML_USE_ENCRYPTED_DESCRIPTOR:
+                defaultParams.setDotcmsSamlUseEncryptedDescriptor(Boolean.parseBoolean(value));
+                break;
+            case DOT_SAML_ACCESS_FILTER_VALUES:
+                defaultParams.setDotSamlAccessFilterValues(value);
+                break;
+            case DOT_SAML_CLOCK_SKEW:
+                try {
+                    defaultParams.setDotSamlClockSkew(Integer.parseInt(value));
+                } catch (Exception ex) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5707914)